src/targets/gpu/fuse_ops.cpp

```diff
@@ -326,6 +328,23 @@ struct find_layernorm
         auto x_ins = r.instructions["x"];
         auto args  = ins->inputs();
 
+        // We dont fuse for non-standard layouts
+        if(not x_ins->get_shape().standard())
+            return;
+
+        auto relements = x_ins->get_shape().lens().back();
+
+        if(relements % 4 == 0)
```

Comment:
this can be simplified to `if(relements > 1024 or (relements % 4 !=0 and relements > 256)) return;`